tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_browser_navigation.py

```diff
@@ -10,52 +9,36 @@
 
 class TestBrowserNavigation(GaiaImageCompareTestCase):
 
-    _community_link_locator = (By.CSS_SELECTOR, '#community a')
-    _community_history_section_locator = (By.ID, 'history')
+    _mozilla_logo_locator = (By.CLASS_NAME, 'main-header')
+    _page_grid_locator = (By.CLASS_NAME, 'promo-grid-wrapper')
 
     def setUp(self):
         GaiaImageCompareTestCase.setUp(self)
         self.connect_to_local_area_network()
 
-        self.test_url = self.marionette.absolute_url('mozilla.html')
+        self.test_url = self.marionette.absolute_url('Mozhome.html')
 
     def test_browser_navigation(self):
         search = Search(self.marionette)
         search.launch()
         browser = search.go_to_url(self.test_url)
-
         browser.switch_to_content()
-        self.verify_home_page()
 
-        Wait(self.marionette).until(lambda m: m.title == 'Mozilla')
+        Wait(self.marionette).until(lambda m: 'building a better Internet' in m.title)
+        Wait(self.marionette).until(lambda m: self.accessibility.is_visible(m.find_element(
+            *self._mozilla_logo_locator)))
+        Wait(self.marionette).until(lambda m: self.accessibility.is_visible(m.find_element(
+            *self._page_grid_locator)))
```

Comment:
You use self.accessibility.is_visible here again, which makes you depending on that library. Why not use is_displayed()?